projects/aqlprofile/src/core/counters.cpp

```diff
@@ -256,6 +256,15 @@ hsa_status_t _internal_aqlprofile_pmc_create_packets(
 
 extern "C" {
 
+PUBLIC_API hsa_status_t aqlprofile_get_version(uint32_t* major, uint32_t* minor, uint32_t* patch,
```

Comment:
The "triplet" implementation in rocprofiler-sdk is quite a bit easier to use, I'd recommend that one